lib/archive_importer.rb

```diff
@@ -37,12 +37,26 @@ def create_user(attr)
     )
     self.user = User.build(data)
     user.save!
+    import_blocks(attr, user)
```

Comment:
This should be moved to the `import` method above (line 13) where all the other stuff is imported to. If the user already exists `create_user` isn't called, so blocks wouldn't be imported.

Reply:
I had to use both - a user could set this on create or on update an existing user